src/lib/common/networks/networks.ts

```diff
@@ -4,6 +4,7 @@ export enum BridgeNetworks {
   Ethereum = "ethereum",
   Polygon = "polygon",
   Avalanche = "avalanche",
+  TRON = 'tron'
```

Comment:
Is Full cap necessary here?  Seems out of place.

Reply:
@Elaixzar should i change it to `Tron`?